ui/src/views/Roles/ViewRole/index.jsx

```diff
@@ -36,20 +36,22 @@ export default class ViewRole extends Component {
   };
 
   handleDeleteRole = async roleId => {
-    this.setState({ error: null, loading: true });
+    this.setState({ dialogError: null, loading: true });
 
-    try {
-      await this.props.client.mutate({
-        mutation: deleteRoleQuery,
-        variables: { roleId },
-      });
+    await this.props.client.mutate({
+      mutation: deleteRoleQuery,
+      variables: { roleId },
+    });
 
-      this.setState({ error: null, loading: false });
+    this.props.history.push(`/auth/roles`);
+  };
 
-      this.props.history.push(`/auth/roles`);
-    } catch (error) {
-      this.setState({ error, loading: false });
-    }
+  handleActionDialogError = error => {
```

Comment:
@helfi92 Nit. What do you think of renaming this to `handleDialogActionError` to keep it consistent with the rest of the function names?

Reply:
Agreed 💯 I will push a patch now.